### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ivan-kadykalo**
[DEMO LINK](https://ivan-kadykalo.github.io/react_static-list-of-posts/)

### CODE REVIEW

File: src/components/PostInfo/PostInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo: React.FC<PreparedPost> = ({
+  title,
+  body,
+  user,
+  comments,
+}) => (
+  <li className="PostInfo" data-cy="post-info">
```
Comment (YegorKochetkov) [inline src/components/PostInfo/PostInfo.tsx:13]: Where is `key={post.id}`?

File: src/components/CommentInfo/CommentInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Comment } from '../../types/types';
+import './CommentInfo.scss';
+
+export const CommentInfo: React.FC<Comment> = ({
+  name,
+  email,
+  body,
+}) => (
+  <li className="CommentInfo">
```
Comment (YegorKochetkov) [inline src/components/CommentInfo/CommentInfo.tsx:10]: Where is `key={comment.id}`?

**Review by YegorKochetkov — CHANGES_REQUESTED** [review]
If I understand correctly, every `li` should have unique key and you forget to add them

File: src/components/PostInfo/PostInfo.tsx (hunk 1 above)
Comment (ivan-kadykalo) [reply src/components/PostInfo/PostInfo.tsx:13]: The keys are used when component are called
image

**Comment by ivan-kadykalo** [comment]
> If I understand correctly, every `li` should have unique key and you forget to add them

No, i use keys when i call my component. And it gets his unique key

File: src/components/PostInfo/PostInfo.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
+.PostInfo {
```
Comment (MaksimusCRS) [inline src/components/PostInfo/PostInfo.scss:1]: Name your classes with small first letter

**Review by MaksimusCRS — APPROVED** [review]

**Pull request closed without merging**